README.md

```diff
@@ -12,23 +12,93 @@ Github CI
+- **kuka_lbr_iiwa_support** contains urdf, config and mesh files for KUKA LBR iiwa robots
+- **kuka_lbr_iiwa_moveit_config** contains configuration files for KUKA LBR iiwa robots necessary for planning with moveit.
 
-## Running the RSI simulator
+## Structure of the support packages
 
+All support packages consist of 4 folders:
+- config: contains joint limits, necessary for time parametrization of trajectories
+- launch: contains launch files to be able to visualize the robot models
+- meshes: contains collision and visual meshes for the robots
+- urdf: contains the xacro files describing the robots, including ros2_control integration (with fake hardware argument)
+
+### Xacro files
+ Each robot has two specific xacro files: a macro ({*robot_name*}_macro.xacro) and another file instantiating this macro ({*robot_name*}.urdf.xacro). Additionally there is a xacro providing ros2_control integration, including the name and type of the hardware interface, hardware parameters and the supported state and command interfaces.
+ Additionally a transmission xacro is provided for gazebo support, but the mechanicalReduction parameters contained within are not valid, only placeholders.
+
+ The macro files contain the links and joints of the main serial chain, including transformations, rotation axes, inertial properties, joint position, velocity and effort limits and the location of the mesh files.
+
+ The macro file follows the ROS-Industrial conventions:
+ - link names are "link_{*i*}"
+ - joint names are "joint_{*i*}"
+ - all link and joint names have a {prefix} argument
+ - base frame: equivalent to the base frame defined by the industrial controller ($ROBROOT)
+ - flange frame: attachment point for EEF models
+ - tool0 frame: all-zeros tool frame
+
+ The frames in the xacros follow the Denavit–Hartenberg conventions of Khalil-Dombre.
+ All robots in the xacros are named according to the following pattern: {kr/lbr_iisy/lbr_iiwa}{*payload*}\_r{*reach*}\_{*version*}.
+ where version is optional and comes from the official product name. (e.g. KR 120 R3100-2 is named kr120_r3100_2)
+
+ The moveit configuration packages also contain xacros, that describe the semantic information of the robots: planning groups, default states and link-pairs, for which collision checking should not be done. The default planning group (from base_link to tool0) is named "manipulator" for all robot arm. An end effector, named "end_effector" is also defined for all robots, which enables visualising end effector paths in rviz.
+
+ To visualise the robot models, the launch files in the launch directory of the support packages can be used. These also start a joint state publisher with default values for all joints, therefore it is not possible to move the robot, only to visualise the frames and joints of the model in default position.
```

Comment:
The initial purpose of these launch files was something like unit testing if the model can be successfully loaded
But as the check for this is not yet implemented, it would indeed make sense to change to joint_state_publisher_gui

Reply:
changed to gui